tests/vmi_configuration_test.go

```diff
@@ -239,6 +239,16 @@ var _ = Describe("[sig-compute]Configurations", func() {
 				vmi = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
 			})
 
+			withMasqueradeBinding := func(vmi *v1.VirtualMachineInstance) {
+				// since we are using it already on a vmi that has masquerade binding
+				// nothing needs to be done
+			}
+
+			withAutoAttachedIfaces := func(vmi *v1.VirtualMachineInstance) {
```

Comment:
Please use regular functions when there is no explicit reason to use a potential closure.

Reply:
Removed the e2e test, so not relevant anymore